src/form/form-wrapper.tsx

```diff
@@ -141,6 +149,8 @@ export const FormWrapper = ({
             "aria-invalid": isInvalidState(),
             "aria-describedby": isInvalidState()
                 ? getErrorTestMessageId()
+                : hasSubtitleLabel()
+                ? getSubtitleId()
                 : undefined,
```

Comment:
`aria-describedby` supports multiple ids. we can concatenate error (takes precedence?) + subtitle together, separated by space https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby#id_reference_list